internal/logging/logger.go

```diff
@@ -148,25 +148,6 @@ func (w *filteredWriterSyncer) Sync() error {
 	return w.dest.Sync()
 }
 
-// UserAwareLoggerMiddleware saves a request-specific logger to the context
-func IdentityAwareMiddleware() gin.HandlerFunc {
-	return func(c *gin.Context) {
-		if val, ok := c.Get("identity"); ok {
-			if id, ok := val.(uid.PolymorphicID); ok {
-				logger := L.With(
-					zapcore.Field{
-						Key:    "id",
-						Type:   zapcore.StringType,
-						String: id.String(),
-					})
-				c.Set("logger", logger)
-			}
-		}
-
-		c.Next()
-	}
-}
-
 // Logger gets the request-specific logger from the context
 // If a request-specific logger cannot be found, use the default logger
 func Logger(c *gin.Context) *zap.Logger {
```

Comment:
This can likely be removed if `logger` isn't being set. The only place it should be called in `errors.go:sendAPIError()`